focus-blocked/index.html

```diff
@@ -140,15 +140,16 @@ <h6>Want to reprogram your brain so you stay on task during focus blocks?
       } else {
         document.getElementById('focusTitle').innerText =
           "Let's keep the focus on " + focus_mode;
-        const endTime = moment(focusEndTime, 'YYYY-MM-DDThh:mm:ss');
+        const endTime = moment(focusEndTime, 'YYYY-MM-DDThh:mm:ss').utc();
         let refreshIntervalId = setInterval(
           () => {
-            const now = moment();
-            if (endTime.diff(now, 'minutes') > 0) {
+            const now = moment().toDate();
+            const nowAsUtc = moment(now.toISOString(), 'YYYY-MM-DDThh:mm:ss').utc()
```

Comment:
I think we can simplify this. No need to create moment objects then convert them to JS dates etc.

![CleanShot 2022-09-04 at 14 00 45](https://user-images.githubusercontent.com/1034890/188296715-df9203bb-65ae-4064-9f38-53741839803e.png)


```
const focusEndTime = '2022-09-04T03:56:51.442Z'
const endTime = moment(focusEndTime);
console.log(endTime.diff(moment(), 'minutes'))
```

Reply:
Changed